taskcluster/docker/inference/Dockerfile

```diff
@@ -27,6 +27,14 @@ RUN apt-get update -qq \
                           libtool \
     && apt-get clean
 
+RUN curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.1/install.sh | bash && \
```

Comment:
Best practice would be to verify the downloaded file against a known checksum (maybe meaningless in this case as the script itself, and then nvm, go off and download more stuff...)

Reply:
No, that's fine. I can compute a checksum and compare it.